sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalogSuite.scala

```diff
@@ -450,6 +451,21 @@ abstract class SessionCatalogSuite extends PlanTest {
     }
   }
 
+  test("alter table add columns") {
```

Comment:
Also add a negative test case for dropping columns, although we do not support it now.